ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/snmp-settings.j2

```diff
@@ -143,6 +143,13 @@ snmp-server location {{ snmp_server.location }}
 {%     if snmp_server.traps.enable is arista.avd.defined(true) %}
```

Comment:
Should we consider allowing the user to disable all traps by default and only enabling the ones interesting under this PR?

Reply:
No, we are considering multiple scenarios. Either the user can enable all and start disabling few or the opposite.

Reply:
I think @xaviramon's point is that we should also have a `no snmp-server enable traps` if it is specifically set as `false`. Normally we try to avoid relying on eos defaults, since those can change or be affected by other commands.

Reply:
Yes, that's my point. We can set the global parameter to true but not to false, even if this is the default value it should also be configurable.

Reply:
OK Agreed we will implement the check for `false` and do `no snmp server enable traps`

Reply:
I got your point. I will implement it accordingly.